cluster/src/main/java/com/hartwig/pipeline/tertiary/orange/Orange.java

```diff
@@ -238,11 +241,14 @@ private List<BashCommand> buildCommands(final SomaticRunMetadata metadata) {
                         "-known_fusion_file",
                         resourceFiles.knownFusionData(),
                         "-ensembl_data_directory",
-                        resourceFiles.ensemblDataCache(),
-                        "-convert_germline_to_somatic");
+                        resourceFiles.ensemblDataCache());
         if (context.equals(Pipeline.Context.RESEARCH)) {
             argumentListBuilder.add("-add_disclaimer");
         }
+        if (!includeGermline) {
+            argumentListBuilder.add("-convert_germline_to_somatic");
+        }
+        metadata.tumor().samplingDate().ifPresent(sd -> argumentListBuilder.add("-experiment_date", sd.toString()));
```

Comment:
I suggest to reformat to `yyMMdd` since any other format will give a WARN in ORANGE